psr/data/uuids/oasys-mapping-uuids.csv

```diff
@@ -361,3 +361,111 @@ ROSH--R1.2.11.2,e03cb9bb-c4aa-4574-85db-c52801fa8460,77980b85-8776-4656-8a3c-a93
+ROSH--R1.2.7.2,e06b0d61-b910-496b-961c-14c8c37ff533,4e55c1fd-6432-4d06-8285-663d07ec932b
+ROSH--R1.2.8.1,df5d635a-6765-42af-9007-6d6d333da5f2,eb468d58-2e58-41af-aab1-20dac4600a75
+ROSH--R1.2.8.2,246bf5f5-b58d-4669-a78f-e47e046ff909,abc903e8-b503-451a-8008-cfc3fcbb4719
+ROSH--R1.2.9.1,c2b221b4-ee1f-41c8-8fc7-9d49998fab35,e282c3fa-901f-442e-b3f8-b125185546dc
+ROSH--R1.2.9.2,4e8ef725-0206-4813-b7c8-21d67c4eb9a6,ea9837ce-ef5a-48e1-8f0e-b23f2089bed2
+ROSH--R1.2.10.1,e887bcea-91d1-4c50-a25e-4335fa1e6ae5,bdbb071d-084d-49a5-8e2e-5df2478718cf
+ROSH--R1.2.10.2,b05f4d19-9663-45fd-a4c0-a64254ad400e,ffafc130-074b-4f3e-9e22-83b16ca2f4b7
+ROSH--R1.2.11.1,66c3bdd2-6f47-4f13-aa7b-39d497d90a89,ce6ed047-4911-4963-9625-b55b017a5c68
+ROSH--R1.2.11.2,78211e3d-1ee1-4c57-aa07-b799c5c0abff,2f82da78-5f87-4d4e-95d1-091f8f74ab63
+ROSH--R1.2.12.1,9692659a-778a-436a-bf4e-fe1924638e37,02c591f8-6d94-4ca0-9753-50be29f72462
+ROSH--R1.2.12.2,795c186d-1b56-439b-97d3-9fc0186b4a1f,2671330b-01f0-4c10-9340-a65d9f2dd8d5
+ROSH--R1.2.13.1,68e31f3a-5175-47e2-986b-d722ad78d893,bea2c828-2f60-4f4a-9d1d-642ba477e36d
+ROSH--R1.2.13.2,f2bf6ac1-7ef1-4dbf-95ae-4c8a1fcbe3a3,d58900a4-80fe-4dc7-8bae-fd0c5e6cc345
+ROSH--R1.2.14.2,39b39e27-3ad3-4e06-a93f-5b5314811379,54cf57ac-eea6-4966-aa5a-64dc7a4b92b8
+ROSH--R1.2.14.2,9662f893-107a-402c-9e23-bba3450e5882,457bc839-720e-4862-9f60-3fa5420a1558
+ROSH--R1.2.10.2,3662710d-ce3e-4e45-bce3-caa4155872aa,df07ac7a-7497-405d-8bdd-8ab753417d42
+ROSH--R1.3.1.2,684b15c0-6b2f-4905-8937-18b637645ade,bca87a71-d996-42af-bea6-2d9ccd6e9bc1
+ROSH--R1.3.2.2,855b585f-7667-49a4-b600-54c2805192d9,c3ebc36f-1520-4e5f-b82c-02f451ddcade
+ROSH--R1.3.3.2,96da74c0-904a-4e7d-b600-7c011dac1149,b7326113-a2ed-4e41-95f0-6f24eec1c97c
+ROSH--R1.3.4.2,9021b522-bc83-40b9-9610-fece250a0149,76642ec1-7f7d-47dd-b202-e48baf57dfda
+ROSH--R1.3.6.2,e3249acc-9b30-4d56-9f7d-842a51a072b4,ee9c7eaf-99af-43e7-bd2a-6d05939eff5f
+ROSH--R1.3.7.2,d1fa0a76-7368-4310-a258-578cd67dfe09,3ec42cfd-901e-434e-abbd-63da928223cb
+ROSH--R1.3.10.2,970476ff-0a9e-4701-86b6-2690978020d0,14ea823e-0ab5-4d48-8cb3-653c2476b6bb
+ROSH--R1.3.12.2,b13eae73-1188-4d6a-a0e8-3e7aee2d6fca,0c56a146-369c-405d-abc2-0c90f7dcbc0d
+ROSH--R1.3.13.2,1c80ffea-3ee2-47e2-9c65-a25d5d2c2391,9cb9d9a6-382d-4dc2-b03c-c8c6bad9e7d4
+ROSH--R1.3.14.2,5b500501-1a79-44ef-b670-c285081ab37a,060c9eee-1b7d-4688-b4d2-fc9329733b59
+ROSH--R1.3.15.2,78bd874f-cf11-43d5-925d-64f9667e1953,af9b7a13-ad65-4370-b0b1-a5d7ce91253e
+ROSH--R1.3.16.2,30a44d90-abcf-43e3-a584-54df1a50539a,b0634bd0-1757-41af-849e-8351244ee442
+ROSH--R1.3.17.2,d2109602-24f4-4a5f-bc46-99d5a48035b2,0036dfc7-7e07-4c26-99a2-a82c632c77ee
+ROSH--R1.3.18.2,3a742a5d-27aa-4c1d-8840-7f0ace66f38a,1ba72aa3-b624-43b2-896f-797c9d59818b
+ROSH--R1.3.19.2,562fdb28-3f01-4bad-b2d9-e65f4ecdfcdc,5ed8b17a-2530-438d-bf57-453f7073cea6
+OFFIN-undefined-family_name,5a26859b-d0fc-42a2-a32f-315c6c2eea1d,e42dc196-2741-49a3-8b39-b58520dd8c75
```

Comment:
Just for completeness - copying in from Slack

```
To try and avoid creating massive SQL diffs (although with only limited success), we keep a cache of the UUIDs we've generated. The uuid files map some key to the uuid. For the oasys mapping, we smash together the section, logical page, question code.

Fixed fields don't have a logical page, so that ends up as undefined
```